packages/vats/test/bootstrapTests/test-liquidation.js

```diff
@@ -0,0 +1,592 @@
+    walletFactoryDriver,
+  } = t.context;
+
+  // price feed logic treats zero time as "unset" so advance to nonzero
+  await advanceTimeBy(1, 'seconds');
+
+  await priceFeedDriver.setPrice(12.34);
+
+  // raise the VaultFactory DebtLimit
+  await governanceDriver.changeParams(
+    agoricNamesRemotes.instance.VaultFactory,
+    {
+      DebtLimit: {
+        brand: agoricNamesRemotes.brand.IST,
+        value: DebtLimitValue,
+      },
+    },
+    {
+      paramPath: {
+        key: {
+          collateralBrand: agoricNamesRemotes.brand.ATOM,
+        },
+      },
+    },
+  );
+
+  // raise the PSM MintLimit
+  await governanceDriver.changeParams(
+    agoricNamesRemotes.instance['psm-IST-USDC_axl'],
+    {
+      MintLimit: {
+        brand: agoricNamesRemotes.brand.IST,
+        value: DebtLimitValue, // reuse
+      },
+    },
+  );
+
+  // confirm Relevant Governance Parameter Assumptions
+  t.like(readLatest('published.vaultFactory.managers.manager0.governance'), {
+    current: {
+      DebtLimit: { value: { value: DebtLimitValue } },
+      InterestRate: {
+        type: 'ratio',
+        value: { numerator: { value: 1n }, denominator: { value: 100n } },
+      },
+      LiquidationMargin: {
+        type: 'ratio',
+        value: { numerator: { value: 150n }, denominator: { value: 100n } },
+      },
+      LiquidationPadding: {
+        type: 'ratio',
+        value: { numerator: { value: 25n }, denominator: { value: 100n } },
+      },
+      LiquidationPenalty: {
+        type: 'ratio',
+        value: { numerator: { value: 1n }, denominator: { value: 100n } },
+      },
+      MintFee: {
+        type: 'ratio',
+        value: { numerator: { value: 50n }, denominator: { value: 10_000n } },
+      },
+    },
+  });
+  t.like(readLatest('published.auction.governance'), {
+    current: {
+      AuctionStartDelay: { type: 'relativeTime', value: { relValue: 2n } },
+      ClockStep: {
+        type: 'relativeTime',
+        value: { relValue: 3n * SECONDS_PER_MINUTE },
+      },
+      DiscountStep: { type: 'nat', value: 500n }, // 5%
+      LowestRate: { type: 'nat', value: 6500n }, // 65%
+      PriceLockPeriod: {
+        type: 'relativeTime',
+        value: { relValue: SECONDS_PER_HOUR / 2n },
+      },
+      StartFrequency: {
+        type: 'relativeTime',
+        value: { relValue: SECONDS_PER_HOUR },
+      },
+      StartingRate: { type: 'nat', value: 10500n }, // 105%
+    },
+  });
+
+  const borrower = await walletFactoryDriver.provideSmartWallet(
+    'agoric1borrower',
+  );
+
+  await borrower.executeOfferMaker(Offers.vaults.OpenVault, {
+    offerId: 'open-vault1',
+    collateralBrandKey,
+    wantMinted: setup.vaults[0].ist,
+    giveCollateral: setup.vaults[0].atom,
+  });
+  t.like(borrower.getLatestUpdateRecord(), {
+    updated: 'offerStatus',
+    status: { id: 'open-vault1', numWantsSatisfied: 1 },
+  });
+
+  await borrower.executeOfferMaker(Offers.vaults.OpenVault, {
+    offerId: 'open-vault2',
```

Comment:
it would be more clear what's common and what's different between opening these 3 vaults if this were a loop over `setup.vaults`

Reply:
done